### PR TITLE
Fix duplicate tag in 044-clk-bufg-pips

### PR DESCRIPTION
**Pull request by tmichalak**
This PR solves issue #875

PIPs with CLK_BUFG_BOT_R_CK_MUXED src wire were solved incorrectly. If the segmatch constraint was -c 1 then it would have been solved correctly, however in order to solve some other 2 bit features in this fuzzer the number of candidates had to be set to 2 (-c 2)

In this PR a separate fuzzer is created for the clk-bufg-muxed PIPs and their calculation is moved from fuzzer 044 to the newly created.

The fix has been tested on the DNM branch (#848)

### CODE REVIEW

File: fuzzers/046-clk-bufg-muxed-pips/generate.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+from prjxray.segmaker import Segmaker
+import os
+import os.path
+import re
+
+
+def bitfilter(frame, word):
+    if frame < 26:
```
Comment (plaes) [inline fuzzers/046-clk-bufg-muxed-pips/generate.py]: `return frame >= 26`

Comment (tmichalak) [reply fuzzers/046-clk-bufg-muxed-pips/generate.py]: This is of course a valid suggestion, but in my opinion the explicit returns make it easier to understand what is filtered out by the bitfilter.

**Comment by tmichalak** [comment]
Red CI on Zynq, will investigate if this is related to this change or another sign of the instability.

**Comment by tmichalak** [comment]
The red CI on Zynq is related to #895.
I am re-running the build, I think the chances for #895 to appear are pretty thin.

**Comment by tmichalak** [comment]
@litghost Do you mean the dbf? I removed the contents on purpose since the PIPs in bits.dbf from fuzzer 044 are not being calculated in this fuzzer, nor are they in the segdata files and therefore appear in the rdb as <const0>. 
BTW when pasting a link could you make sure it's through ``console.cloud.google.com`` rather than ``cnsviewer2.corp.google.com`` - I don't have access to the latter.

**Comment by litghost** [comment]
> @litghost Do you mean the dbf?

Ya, dbf.  If you look at the segbits diff, the new output is missing some of the negative bits defined by the dbf, so it was required and you are no longer using it.

**Comment by litghost** [comment]
Please rebase and lets re-run.

**Pull request merged**